Automattic/pocket-casts-ios
# Improve folders performance

Fixes #

This PR implements a folder cache of it's top podcasts uuids

## To test

- Start app using a large database with folders with lot of podcasts episodes
- Open podcasts view
- Scroll down ( still some slow down can happen while the cache is being fulfilled)
- Scroll back up and scroll performance should be optimal.
- Open a folder
- Change folder properties like: sort order, name, color
- Go back to the root level
- Check if folder image is updated
- Search in the podcasts view for some folder name, see if the results cells show the folders correctly

## Checklist
